go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/lufia/iostat v1.2.1
 	github.com/mattn/go-xmlrpc v0.0.3
 	github.com/mdlayher/wifi v0.0.0-20200527114002-84f0b9457fdd
-	github.com/prometheus/client_golang v1.11.0
+	github.com/prometheus/client_golang v1.11.1
```

Comment:
```suggestion
	github.com/prometheus/client_golang v1.12.1
```

Reply:
That's probably fine at this point. We can bump the minimum version in `go.mod`.